src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/AbstractMapper.php

```diff
@@ -154,6 +157,16 @@ protected function addFraudCheckData(
         return $requestTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return string
+     */
+    protected function getGender(QuoteTransfer $quoteTransfer): string
+    {
+        return static::GENDER_MAPPING[$quoteTransfer->getCustomer()->getSalutation()];
```

Comment:
I would add a check for existing salutation just in case.

Reply:
Check has been added.